weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/tsDataModelHooks.ts

```diff
@@ -325,22 +328,24 @@ const useCallsNoExpansion = (
         result: [],
       };
     } else {
-      allResults.forEach(call => {
-        callCache.set(
-          {
-            entity,
-            project,
-            callId: call.callId,
-          },
-          call
-        );
-      });
+      if (!columns) {
```

Comment:
we can't cache results if the query had a specific column request, as future queries might not have the same columns. We could add the columns to the cache key, but then we invalidate our deletion from cache scheme. Almost always (currently), specifying columns makes the query fast, so caching is less important. Not a longterm solution.

Reply:
very good! i was wondering if this PR would account for that - nice (:

Reply:
would be nice to add this as a comment